### PR TITLE
fix: #8 - use fractional seconds on php 7.2

### PR DESCRIPTION
**Pull request by bytestream**
This PR works around https://bugs.php.net/bug.php?id=76386:
1. Use the Laravel query builder to insert new messages into the database, with fractional second precision. 
2. Use Laravel Eloquent to obtain messages from the database. Precision is not defined on the Eloquent model so, fractional seconds are lost. This is OK however so long as `orderBy('created_at')` is used in the SQL query to ensure data is ordered by the sql server before it's sent to the client. I don't think we have a current use case for viewing microsecond precision. We just want the correct order of messages.

It's not possible to use Eloquent for both insertion and selection. The `getDateFormat()` means timestamps are cast to the specified format. On PHP 7.2, fractional seconds or microseconds are not returned from the database and so Carbon complains that data is missing when casting the data.

TLDR; use the query builder if you need to insert fractional seconds. Have Eloquent pretend fractional second precision is not defined.

### CODE REVIEW

**Comment by jshah4517** [comment]
This doesn't work on PHP 7.3+, Carbon gives an error as it expects data in normal date time format without microseconds but Eloquent provides it with microseconds.
